<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogErgo</class>
 <widget class="QDialog" name="DialogErgo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditTesfuerzo"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEditMets"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelVTDVI">
     <property name="text">
      <string>METS</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="pushButtonErgoGuardar">
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="dateEditErgo"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelDilatVi">
     <property name="text">
      <string>Tiempo de esfuerzo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Fecha</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogErgo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogErgo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
